### ISSUE TITLE
[Usable – Make Code – Print] Visual focus indicator is not visible proper on the control. 

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3774]
User Experience:
If, the visual focus indicator is not visible on the control, then the user who depends upon keyboard for navigation might get affected hence failed to identify the current focus on the control.

Note: User credentials should NOT be included in the bug.

Repro steps:

1. Launch application URL
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Settings" option appearing in header section and select it.
6. Navigate to "Print" option and select it.
7. Navigate through all the controls appearing under "Print Code" page
8. Verify the issue.

Actual Result:
Visual focus indicator is not visible properly on control like : More setting. control is getting underline when focus moves on it.

Expected Result:
Visual focus indicator should be visible properly on control. So, that the keyboard user does able to see the perform its action on the control properly.

Note:

This issue does not exists with Firefox and Chrome Browser.

Test Environments:
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0)
Screen Reader: Narrator

**Comment by Shalini3457** [comment]
Labels:
#Accessibility; #A11yUsable; #A11ySev4; #HCL; #HCL-MakeCode-New; #Win10-Edge(Chromium)

![Usable](https://user-images.githubusercontent.com/72871089/100340826-c5689c80-3001-11eb-8046-2cb25b88001a.png)

**Comment by jwunderl** [comment]
This is the browsers print dialog, we do not have control over this.
